targets/mgyamlfmt/targets.go

```diff
@@ -0,0 +1,40 @@
+package mgyamlfmt
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"go.einride.tech/mage-tools/mgtool"
+	"gopkg.in/yaml.v3"
+)
+
+func FormatYaml() error {
+	return filepath.Walk(mgtool.GetGitRootPath("."), func(path string, _ os.FileInfo, err error) error {
```

Comment:
Should we support excluding directories ? Right now im just defaulting to the git root